index.html

```diff
@@ -2570,6 +2604,151 @@ <h3>
           </pre>
         </div>
       </section>
+      <section>
+        <h2><a id="icon-masks"></a>
+          Icon masks and safe zone
+        </h2>
+        <p>
+          Some platforms have their own preferred icon shape, but as web
+          applications should work across multiple platforms, it is possible to
+          indicate that an icon can have a user-agent-specified mask applied by
+          adding the <a>maskable</a> purpose. This allows the platform to ensure
+          that the icon looks well integrated with the platform, and even apply
+          different masks and background colors in different places throughout
+          the platform.
+        </p>
+        <p>
+          The <dfn>safe zone</dfn> is the area within a <a>maskable</a> icon
+          which is guaranteed to always be visible, regardless of user agent
+          preferences. It is defined as a circle with center point in the center
+          of the icon and with a radius of 2/5 (40%) of the icon size, which means
+          the smaller of the icon width and height, in case the icon is not
+          rectangular.
+        </p>
+        <p class="note">
+          Designers of <a>maskable</a> icons will want to make sure that all
+          important parts are within the <a>safe zone</a>.
+        </p>
+        <figure>
+          <img src="images/safe-zone.svg" width="340" height="340" alt=
+          "safe zone illustrated">
+          <figcaption>
+            The safe zone is a centrally positioned circle, with radius 2/5
+            (40%) of the minimum of the icon's width and height.
+          </figcaption>
+        </figure>
+        <p>
+          All pixels in this zone are guaranteed to be seen in all masks.
+          Pixels outside the safe zone are not guaranteed to (but can) be visible
+          depending on the applied mask.
+        </p>
+        <p>
+          The user agent MAY apply a mask of any size, making any pixels that
+          are more than 2/5ths of the image size (minimum of width and height
+          if non-square) away from the center (the <a>safe zone</a>)
+          transparent.
+        </p>
+        <p>
+          The user agent MUST NOT make any pixel within the safe zone
+          transparent.
+        </p>
+        <p>
+          The user agent MAY enlarge the icon by adding additional padding.
+        </p>
+        <p>
+          If the icon contains transparent pixels, the user agent MUST composite
+          the icon onto a color (eg. white) of the user agent's choice.
```

Comment:
nit: The word "solid" was dropped ("solid color"). Maybe you thought it was unnecessary. I don't mind either way.